src/main/java/com/nekokittygames/mffs/common/MFFSRecipes.java

```diff
@@ -41,7 +42,9 @@ public static void AddIC2Recipes() {
 		Recipes.macerator.addRecipe(Recipes.inputFactory.forStack(new ItemStack(
 				ModBlocks.MONAZIT_ORE, 1)),new NBTTagCompound(),false,new ItemStack(ModItems.FORCICIUM, MONAZIT_MACERATION_OUTPUT));
 
-        Recipes.matterAmplifier.addRecipe(Recipes.inputFactory.forStack(new ItemStack(ModItems.FORCICIUM)),5000,new NBTTagCompound(),false);
+		if (!Loader.isModLoaded("ic2-classic-spmod")){
```

Comment:
Shouldn't this be being fixed in Classic rather than being special cased in MFFS?

Reply:
it's temporary as who knows how long till it gets fixed.

Reply:
Could at least do with a comment referring to a Classic issue about it, it'll be forgotten about and never removed otherwise.

Reply:
Sure I can do that

Reply:
Actually never mind I may just remove it as the issue will be fixed in the next ic2 classic release